### ISSUE TITLE
The export of Midifile destroys aftertouch events

### ISSUE THREAD

**Issue by AndrewPy** [muse-sequencer/muse#531]
If you create a song (or import a Midifile) with aftertouch events and export it to a Midifile, then the aftertouch events turn into RPN events (in the exported Midifile).

![1_aftertouch_before](https://cloud.githubusercontent.com/assets/20823006/17745848/d14109e4-64ad-11e6-8e1b-6f497e6b68de.png)

![2_aftertouch_after](https://cloud.githubusercontent.com/assets/20823006/17745860/dcf74d84-64ad-11e6-8693-a03340dc04e9.png)

**Comment by terminator356** [comment]
On Wednesday, August 17, 2016 10:08:15 AM EDT AndrewPy wrote:
> If you create a song (or import a Midifile) with aftertouch events and
> export it to a Midifile, then the aftertouch events turn into RPN events.
> 
> ![1_aftertouch_before](https://cloud.githubusercontent.com/assets/20823006/1
> 7745848/d14109e4-64ad-11e6-8e1b-6f497e6b68de.png)
> 
> ![2_aftertouch_after](https://cloud.githubusercontent.com/assets/20823006/17
> 745860/dcf74d84-64ad-11e6-8693-a03340dc04e9.png)

Hi, quick question before I test here, have you tried the latest git master?

Recently (month ago) I fixed two things in there concerning aftertouch export.
IIRC, it wasn't exporting them at all, I think.
Maybe I didn't get something right... I could swear I tested it... 
I will check.

Tim.

**Comment by AndrewPy** [comment]
> Hi, quick question before I test here, have you tried the latest git master?

Sorry - no, I have tried, but I had problems to compile it under debian.

**Comment by terminator356** [comment]
On Wednesday, August 17, 2016 1:10:20 PM EDT AndrewPy wrote:
> > Hi, quick question before I test here, have you tried the latest git
> > master?
> Sorry - no, I have tried, but I had problems to compile it under debian.

Git master tests OK here.
Both aftertouch (channel pressure) and poly-aftertouch (key pressure)
 exported and then imported to QTractor fine.

I think the old bad behaviour may have made them RPNs.
It was that way from the beginning. I just noticed last month.

But my fault for missing this small section when I added
 aftertouch controller support years ago.

You definitely need the latest git master.

Tim.

**Comment by AndrewPy** [comment]
> You definitely need the latest git master.

I tried it again. 

Unfortunately I had to disable nearly all features in the cmake options to compile the the latest git master under debian stable, but now the midifile export works fine. 

Thank you!

**Comment by AndrewPy** [comment]
Sorry, I only wanted to test the midifile export and made no deep search for the library problems. 

For now I have removed the whole bunch of qt5 libraries and will use muse 2.2.1 again. (Maybe next time I will try muse 3.0 it in a virtual machine.)

But even after disabling LV2 and DSSI there was one strange problem with moc_lv2host.cpp: 81:16: error: "MusECore::LV2PluginWrapper_Window" has not been declared. (I had to comment it out to compile muse.)

Andrew
